136.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var singleNumber = function (nums) {
+  let arr = nums.sort((a, b) => a - b);
+
+  for (let i = 0; i <= nums.length - 1; i += 2) {
+    if (arr[i] !== arr[i + 1]) {
+      return arr[i];
+    }
+  }
+
+  return arr[arr.length - 1];
```

Comment:
```suggestion
    for (let i = 0; i <= nums.length; i += 2) {
        if (arr[i] !== arr[i + 1]) {
            return arr[i];
        }
    }
```
You can also do this